test/support/assertions.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Test
+  module Unit
+    module Assertions
```

Comment:
From my experience, we normally don't add project-specific helpers to a library's modules. We'd do this instead:

```ruby
module DEBUGGER__
  module AssertionHelpers
    include Test::Unit::Assertions # this may not be necessary

    def assert_line_num(expected)
      # .....
    end

# in test_case.rb

module DEBUGGER__
  class TestCase < Test::Unit::TestCase
    include TestUtils
    include AssertionHelpers

```

Reply:
@st0012 
Thanks for giving me information, but I'd like discuss more. Project-specific helpers are added to a library's modules in some ruby projects. What do you think?

https://github.com/ruby/irb/blob/master/test/lib/test/unit/core_assertions.rb
https://github.com/ruby/ruby/blob/master/tool/lib/test/unit/core_assertions.rb

Reply:
If we run tests of different projects in a process, it is a bit safer to isolate the definitions.

Reply:
Ok, that way sounds better. I changed it.